packages/orchestration/README.md

```diff
@@ -1,14 +1,14 @@
 # @sap-ai-sdk/orchestration
 
-This package incorporates generative AI orchestration capabilities into your AI activities in SAP AI Core and SAP AI Launchpad.
+This package provides generative AI orchestration capabilities, integrating seamlessly with SAP AI Core and SAP AI Launchpad to enhance your AI workflows.
```

Comment:
[q] For me the usage of workflow here is a bit confusing. Orchestration is not really related to the whole ML worfklow scenario , is it?

Reply:
Reverted to use the old sentence.